crates/sui-core/src/checkpoints/checkpoint_executor/mod.rs

```diff
@@ -64,7 +64,7 @@ pub(crate) mod tests;
 type CheckpointExecutionBuffer = FuturesOrdered<JoinHandle<VerifiedCheckpoint>>;
 
 /// The interval to log checkpoint progress, in # of checkpoints processed.
-const CHECKPOINT_PROGRESS_LOG_COUNT_INTERVAL: u64 = 5000;
+const CHECKPOINT_PROGRESS_LOG_COUNT_INTERVAL: u64 = 10;
```

Comment:
The 5000 interval is meant to avoid being noisy during catchup processing 30 ~ 100 checkpoints per sec. Just curious, why is this log message useful when the node has fully caught up?

Reply:
I was checking this log when investigating this [issue](https://mysten-labs.slack.com/archives/C03AW7J5Q5U/p1684163688124319) but could hardly find any logs. I'm not sure why we worry about 30~100 checkpoint per sec logging rate:
1. catching up from scratch is a less common scenario compared to sync when up-to-date
2. we have tons of other logs for each tx, so a few log lines for every 10 checkpoint seems negligible

Reply:
There are only debug logs for each tx. If you think it is going to be useful, how about logging every checkpoint in debug mode?

Reply:
I didn't realize it's info logging! Yeah I think it makes sense. I will make the change